### PR TITLE
Miscellaneous clean-up

### PR DESCRIPTION
**Pull request by dwang20151005**
I'd like not to maintain such a trivial fork of ocaml-re, so I'm sending upstream all these cosmetic changes.  If anything in this PR violates your local style, just tell me, and I'll revert them internally.  Thanks!

### CODE REVIEW

File: lib_test/fort_unit.ml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 (* ounit compatibility layer for fort tests *)
-open OUnit2
+open OUnit
```
Comment (Drup) [inline lib_test/fort_unit.ml]: Why ?

File: lib/re_glob.mli [hunk 2]
```diff
@@ -54,7 +54,7 @@ val glob :
     slash. Defaults to true.
 
     If [expand_braces] is true, braced sets will expand into multiple globs,
-    e.g. a\{x,y\}b\{1,2\} matches axb1, axb2, ayb1, ayb2.  As specified for bash, brace
+    e.g. a{x,y}b{1,2} matches axb1, axb2, ayb1, ayb2.  As specified for bash, brace
```
Comment (Drup) [inline lib/re_glob.mli]: That's needed for ocamldoc, iirc.

File: lib_test/test_str.ml [hunk 3]
```diff
@@ -6,8 +6,6 @@ let eq_match ?pos ?len ?(case = true) r s =
     ~printer:arr_ofs_printer
     (fun () ->
        let pos = match pos with None -> 0 | Some p -> p in
-       let pat = if case then Str.regexp r else Str.regexp_case_fold r in
-       let _s_start = Str.search_forward pat s pos in
```
Comment (Drup) [inline lib_test/test_str.ml]: Why ?

**Comment by Drup** [comment]
Please cleanup your commit log.

File: lib/re_glob.mli (hunk 2 above)
Comment (rgrinberg) [inline lib/re_glob.mli]: Indeed, ocamldoc build will fail if you don't escape these.

File: lib_test/test_str.ml (hunk 3 above)
Comment (dwang20151005) [inline lib_test/test_str.ml]: These values are unused.  Should they be kept around for some reason?

**Comment by dwang20151005** [comment]
Thanks for being patient with a new git/github user.  I hope the squashing is the cleanup you wanted?

**Comment by dwang20151005** [comment]
Ping?

File: lib_test/test_str.ml (hunk 3 above)
Comment (rgrinberg) [inline lib_test/test_str.ml]: Pretty sure this isn't semantically equivalent however. `search_forward` raises when a result isn't found.

So I guess `ignore (Str.search_forward ..)` is equivalent. What do you think?

File: lib_test/pcre_scan.ml [hunk 4]
```diff
@@ -3,6 +3,6 @@ let x = Re_pcre.regexp "aa?b"
 let _ =
   let s = String.make (1024*1024) 'a' in
   s.[1024*1024-1] <- 'b';
-  for i = 0 to 99 do
+  for _i = 0 to 99 do
```
Comment (rgrinberg) [inline lib_test/pcre_scan.ml]: Just `_` doesn't work here?

File: lib_test/scan.ml [hunk 5]
```diff
@@ -145,6 +145,6 @@ let exec s = loop {i_cols = cols; last = String.length s} s 0 cols st1 st1
 let _ =
   let s = String.make (1024*1024) 'a' in
   s.[1024*1024-1] <- 'b';
-  for i = 0 to 99 do
+  for _i = 0 to 99 do
```
Comment (rgrinberg) [inline lib_test/scan.ml]: ditto

**Comment by Drup** [comment]
I appreciate the effort, really. :)

However, those changes don't really have any point. They don't even reduce the number of warnings and don't improve the code quality. On top of that, you remove bits of code for no reason (and pretty sure those are necessary, since Str is stateful ..)

@rgrinberg Please close.

**Comment by rgrinberg** [comment]
Yes I don't think these are necessary, also I have no clue why the oasis files is different.

**Comment by dwang20151005** [comment]
Okay, I will revert internally.  Thank you for taking the time to review.

**Pull request closed without merging**